### DISCUSSION TITLE
PyCon Niger 2024 support and sponsorship

### DISCUSSION THREAD

**Discussion by Yusee-Programmer** [community/community#126576] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Call to Sponsor PyCon Niger 2024!

if you're interested in sponsoring OSS tech conference like PyCon,
feel free to contact us and talk more about this opportunity.

[redacted by moderator]

**Comment by ebndev** [comment, marked as answer]
Hi @Yusee-Programmer, thanks for being a part of the GitHub Community!

However, we currently do not allow self-promotion on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Please keep in mind that the Sponsors category is dedicated for the GitHub Sponsors program. We appreciate your understanding!
